dev/run-tests-jenkins.py

```diff
@@ -181,8 +181,8 @@ def main():
     short_commit_hash = ghprb_actual_commit[0:7]
 
     # format: http://linux.die.net/man/1/timeout
-    # must be less than the timeout configured on Jenkins (currently 350m)
-    tests_timeout = "300m"
+    # must be less than the timeout configured on Jenkins (currently 400m)
```

Comment:
Updated by http://apache-spark-developers-list.1001551.n3.nabble.com/build-system-bumped-pull-request-builder-job-timeout-to-400mins-td24647.html